c2corg_ui/static/js/lang.js

```diff
@@ -70,7 +74,7 @@ app.LangController = function(gettextCatalog, langUrlTemplate,
    */
   this.lang = this.cookies_.get('interface_lang') ||
       ngeoGetBrowserLanguage(this['langs']) || 'fr';
-  this.updateLang();
+  this.updateLang(null);
```

Comment:
I think you can avoid passing a ``null`` argument by adding a default value in the ``updateLang`` definition.
For instance
```
app.LangController.prototype.updateLang = function(lang) {
  lang = lang || null;
```